src/test/regress/expected/multi_modifications.out

```diff
@@ -317,6 +317,11 @@ WITH deleted_orders AS (INSERT INTO limit_orders DEFAULT VALUES RETURNING *)
 UPDATE limit_orders SET symbol = 'GM';
 ERROR:  cannot perform distributed planning for the given modification
 DETAIL:  Common table expressions are not supported in distributed modifications.
+-- updates referencing a column are supported
+UPDATE limit_orders SET bidder_id = bidder_id + 1 WHERE id = 246;
```

Comment:
We can maybe add a few more tests here, e.g. a function call. Also, it's worth checking whether the table was correctly updated.